### PR TITLE
View timeline delay 2

### PR DESCRIPTION
**Pull request by kevers-google**

### CODE REVIEW

File: src/scroll-timeline-base.js [hunk 1]
```diff
@@ -449,45 +449,61 @@ function range(timeline, phase) {
...
+  // 0% progress represents the position at which the start border edge of the
+  // element’s principal box coincides with the end edge of its view progress
+  // visibility range.
+  // 100% progress represents the position at which the end border edge of the
+  // element’s principal box coincides with the start edge of its view progress
+  // visibility range.
   const coverStartOffset = viewPos - containerSize + inset.end;
   const coverEndOffset = viewPos + viewSize - inset.start;
+
+  // Contain:
```
Comment (mehdi-kazemi) [inline src/scroll-timeline-base.js:464]: Well-defined, and is it also backed by spec?

Comment (bramus) [reply src/scroll-timeline-base.js:464]: A recent spec updated defined all these: https://drafts.csswg.org/scroll-animations-1/#view-timelines-ranges

**Comment by bramus** [comment]
Confirming that this one fixes https://github.com/flackr/scroll-timeline/issues/73

**Comment by bramus** [comment]
Confirming that this one fixes https://github.com/flackr/scroll-timeline/issues/62 as well

**Review by mehdi-kazemi — APPROVED** [review]

**Pull request merged**